scripts.py

```diff
@@ -606,9 +606,15 @@ class LaneSweeperScript(LibraryInputScript):
     """Do something to each lane in a library."""
 
     def process_library(self, library):
-        queue = self._db.query(Lane).filter(
+        from lane import WorkList
+        top_level_lanes = self._db.query(Lane).filter(
             Lane.library==library).filter(
                 Lane.parent==None).all()
+        top_level = WorkList()
+        top_level.initialize(
+            library, library.name, children=list(top_level_lanes)
+        )
+        queue = [top_level]
```

Comment:
Without this code, running the script that makes OPDS grouped feeds will make grouped feeds for 'fiction', 'nonfiction', etc, but it won't make the top-level grouped feed.